net-libs/nativebiginteger/nativebiginteger-0.9.42.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 2018-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit java-pkg-2 toolchain-funcs
+
+DESCRIPTION='jbigi library used by net-vpn/i2p'
```

Comment:
While this might work okay, why wouldn't you use `" "`?

Reply:
I works the same, IMO it explicits that there will not be variables references or other bash specifics.

Reply:
You have mixed `' '` and `" "` throughout these ebuilds. Stick to one style, and let that style be what we normally use, and what is used in skel format. So `" "`.